bin/openstack-install

```diff
@@ -193,18 +193,6 @@ if __name__ == '__main__':
             print('Only supports killing a single install for now.')
             sys.exit(1)
 
-    if os.path.isfile(os.path.join(cfg.cfg_path, 'installed')):
-        msg = ("\n\nError:\n\n"
-               "Previous installation detected. Did you mean to run "
-               "openstack-status instead? \n"
-               "If attempting to re-install please remove "
-               "~/.cloud-install or move it to a different \n"
-               "directory name (eg ~/cloud-install-old). "
-               "Otherwise you may wish to uninstall with:\n\n"
-               "    $ sudo openstack-install -u\n\n")
```

Comment:
Is there any way we can keep this check around? 
What happens if you run openstack-install twice with this branch?